packages/fleather/example/lib/autoformat_iterator.dart

```diff
@@ -0,0 +1,364 @@
+import 'dart:convert';
```

Comment:
It might be better to have an example app showcasing every feature we have instead of a separate example for each feature, and a simple example to get users started quickly. I suggest to remove this example from this PR and create another PR for a complete advanced example. WDYT?

Reply:
Hi @Amir-P,

This is really the main example with some additions to showcase this extra feature so it could overwrite the example app you have there already if you choose to use it. I have been taking notes with the intention to eventually help with some of the documentation and covering several use cases with Fleather and Parchment. Trying to build a few cohesive examples. This and https://github.com/fleather-editor/fleather/pull/371 both have examples working with the same "youtube" custom block example. 

However, I'm not married to any of it! Definitely take what you like and leave the rest. :)

Reply:
In that case let's rename this into something like `example_advanced.dart`, then we can simplify current example we have in another PR by removing inline images and embeds for users to get started with basics quicker. Also I can see you've defined `YoutubeEmbed` but never used it.

Reply:
Sounds good to me. I also realized I left the old version of the autoformats.buildWithFallback command in the example. Let me clean it up and post a new commit. Should be able to do it closer to the weekend -- need to wrap up a work project tomorrow.